products/chef-client.md

```diff
@@ -0,0 +1,107 @@
+---
+title: Chef Infra Client
+category: server-app
+iconSlug: chef
+permalink: /chefinfraclient
+tags: ruby-runtime
+alternate_urls:
+-   /chefclient
+releasePolicyLink: "https://docs.chef.io/versions/"
+changelogTemplate: "https://docs.chef.io/release_notes_client/#{{'__LATEST__'|replace:'v',''}}/"
+versionCommand: chef-client -v
+releaseDateColumn: true
+activeSupportColumn: true
+
+auto:
+-   git: https://github.com/chef/chef/
```

Comment:
This is the same problem as the comment I left on the infra server PR. Versions in GitHub do not match the releases and Chef doesn't use GitHub's releases feature either. You have to query their Omnitruck API to get stable release versions.

https://docs.chef.io/api_omnitruck/

The versions in GitHub result in a build, that if successful, goes to the current channel. Think of those as betas.

Reply:
As the problem on this PR is the same on #4425 , I'm passing it to draft, while we find a solution for #4425 

Once we find a solution, I will apply it here as well